File: fmLaPaz@moji.physics/get_artist.rb
#!/usr/bin/ruby
require 'rubygems'
require 'json'

fmLaPazJson = File.read('/home/diego/.local/share/gnome-shell/extensions/fmLaPaz@moji.physics/fmLaPaz_now.json')
obj = JSON.parse(fmLaPazJson,{allow_nan:true})
info=[]

#obj.Results.map{|result| result.Paths.map{|path| info << path.Domain }}
puts obj["data"][0]["song"]

File: fmLaPaz@moji.physics/get_artist.py
#!/usr/bin/python3.9
import sys,json
sourceFile=sys.argv[1]
with open(sourceFile,'r') as myFile:
    data=myFile.read()
myFile.close()

info_data=json.loads(data)
dat=[]
for item in info_data['data']:
    dat.append(item)

print(dat[0]['listeners'],",",dat[0]['song'])
